projects/plugins/jetpack/tests/e2e/helpers/sync-helper.js

```diff
@@ -0,0 +1,53 @@
+import { execWpCommand } from 'jetpack-e2e-commons/helpers/utils-helper.cjs';
+import logger from 'jetpack-e2e-commons/logger.cjs';
+
+export async function enableSync() {
+	return await execWpCommand( 'jetpack sync enable' );
+}
+
+export async function disableSync() {
+	return await execWpCommand( 'jetpack sync disable' );
+}
+
+export async function getSyncStatus() {
+	return await execWpCommand( 'jetpack sync status' );
+}
+
+export async function enableDedicatedSync() {
+	return await execWpCommand( 'option update jetpack_sync_settings_dedicated_sync_enabled 1' );
+}
+
+export async function disableDedicatedSync() {
+	return await execWpCommand( 'option update jetpack_sync_settings_dedicated_sync_enabled 0' );
+}
+
+export async function isSyncQueueEmpty() {
+	const status = await getSyncStatus();
+
+	return status.includes( 'queue_size' ) && status.includes( 'queue_size	0' );
+}
+
+/**
+ * Wait till the Sync Queue is empty or a certain timeout has been reached.
+ *
+ * @param {number} [timeoutMs=10000] - The timeout to stop checking if the Sync Queue is empty.
+ */
+export async function waitTillSyncQueueisEmpty( timeoutMs = 10000 ) {
```

Comment:
@adimoldovan If needed, in the future, this could be refactored as a generic `waitTill` function in common utils. I didn't find any similar functionality in place, please let me know if there is.
The concept is a function that will wait until a certain condition is met or a timeout has been reached.

Reply:
just found `page.waitForFunction()`: https://playwright.dev/docs/api/class-page#page-wait-for-function

I don't think it makes sense to rewrite it just yet. But maybe note it for future us when/if we decide to refactor it into a more generic function.